scapy/contrib/ltp.py

```diff
@@ -140,8 +141,17 @@ class LTP(Packet):
         #
         ConditionalField(SDNV2("CheckpointSerialNo", 0),
                          lambda x: x.flags in _ltp_checkpoint_segment),
-        ConditionalField(SDNV2("ReportSerialNo", 0),
-                         lambda x: x.flags in _ltp_checkpoint_segment),
+        MultipleTypeField([
+            (SDNV2("ReportSerialNo", 0),
```

Comment:
Now that I see, you should actually use a single `ConditionalField` because the two fields are identical.

Reply:
Are you saying I should use one ConditionalField with a lambda funciton like this:
`lambda x: x.flags in _ltp_checkpoint_segment or x.flags == 8`

Reply:
Yes 👍